README.md

```diff
@@ -9,6 +10,9 @@ Displays the tasks and assignments that are ongoing.
 
 # Setup and running locally
 
+
+Firstly setup [Volta](https://docs.volta.sh/guide/getting-started) | [Why Volta?](https://docs.volta.sh/guide/#why-volta)
```

Comment:
nitpick: This can use a `Recommended` sub-heading, since one can run the project without volta but we (contributors and app owners) prefer this tooling.

Reply:
@TanishqSingla adding `Recommended` will not be the right word, we don't want it to be a good-to-have feature, rather it should be the default.